tools/http_benchmark.py

```diff
@@ -60,15 +60,32 @@ def hyper_http_benchmark(hyper_hello_exe):
 
 def http_benchmark(deno_exe, hyper_hello_exe, core_http_bench_exe):
     r = {}
+    m = {}
+
     # TODO Rename to "deno_tcp"
-    r["deno"] = deno_http_benchmark(deno_exe)
-    r["deno_net_http"] = deno_net_http_benchmark(deno_exe)
-    r["deno_core_single"] = deno_core_single(core_http_bench_exe)
-    r["deno_core_multi"] = deno_core_multi(core_http_bench_exe)
-    r["node"] = node_http_benchmark()
-    r["node_tcp"] = node_tcp_benchmark()
-    r["hyper"] = hyper_http_benchmark(hyper_hello_exe)
-    return r
+    denoHttp = deno_http_benchmark(deno_exe)
+    denoNetHttp = deno_net_http_benchmark(deno_exe)
+    denoCoreSingle = deno_core_single(core_http_bench_exe)
+    denoCoreMulti = deno_core_multi(core_http_bench_exe)
+    nodeHttp = node_http_benchmark()
+    nodeTcp = node_tcp_benchmark()
+    hyperHttp = hyper_http_benchmark(hyper_hello_exe)
+
+    dic = {
+        "deno": denoHttp,
```

Comment:
you don't need to create these temp variables, why not use the dict comprehension:
```
return {
  "deno": deno_http_benchmark(deno_exe)
  ...
}
```
that seems a little cleaner.

Reply:
thanks! it does look much cleaner